state/interface.go

```diff
@@ -113,10 +113,13 @@ type UnitsWatcher interface {
 	WatchUnits() StringsWatcher
 }
 
-// ModelMachinesWatcher defines a single method -
+// ModelMachinesWatcher defines a the methods required for listening to
+// machine lifecycle events or a combination of lifecycle events and changes
+// to the agent start time field.
 // WatchModelMachines.
 type ModelMachinesWatcher interface {
```

Comment:
I wouldn't necessarily move it in this patch, but this is only ever used in `apiserver` and sub-packages. It would be better style for it to reside there instead of `state`.